alot/db/message.py

```diff
@@ -107,13 +107,11 @@ def get_thread_id(self):
         return self._thread_id
 
     def get_message_parts(self):
-        """returns a list of all body parts of this message"""
+        """yield all body parts of this message"""
         # TODO really needed? email  iterators can do this
```

Comment:
I'm with @lucc on this. we'd have to call email.iterators.typed_subpart_iterator multiple times (since mainpart doesn't take a `*` argument).